pypeman/commands.py

```diff
@@ -179,6 +161,15 @@ def main(debug_asyncio=False, profile=False, cli=False, remote_admin=False):
 
     print("End started tasks...")
 
+    # TODO: Shouldn't we also stop all endpoints?
+    # TODO: and if yes, before or after stopping the channels
+    # Intuitively, as we start endpoints after the channels we should probably stop
+    # the endpoints before te channels
+
+    # for end in endpoints.all_endpoints:
+    #     loop.run_until_complete(end.start())
```

Comment:
```suggestion
    #     loop.run_until_complete(end.stop())
```

Reply:
oops already fixed in another development branch. will do.

Reply:
done